pyproject.toml

```
@@ -30,7 +30,7 @@ requires-python = ">=3.8"
 description = "A CalVer version manager that supports the future."
 readme = "README.rst"
 dependencies = [
-    "setuptools >= 61.0",
+    "packaging",
```

Just to make sure that we get a version that has post and dev attributes  . it looks like they were added in 17.0

I know that this is a long time ago... so maybe it's not needed

https://github.com/pypa/packaging/blob/main/CHANGELOG.rst#170---2017-02-28

```suggestion
    "packaging >= 17.0",
```

---

That's an excellent idea, thanks!